selfdrive/car/volkswagen/values.py

```diff
@@ -24,6 +25,9 @@ class CANBUS:
   pt = 0
   cam = 2
 
+TRANS = car.CarParams.TransmissionType
+GEAR = car.CarState.GearShifter
```

Comment:
```suggestion
TransmissionType = car.CarParams.TransmissionType
GearShifter = car.CarState.GearShifter
```

this is the convention used throughout the codebase

Reply:
Done!